app/controllers/token_auth/api/payloads_controller.rb

```diff
@@ -53,7 +53,7 @@ def find_token!
 
         return if @authentication_token
 
-        fail TokenAuth::Concerns::ApiResources::Api::Unauthorized
+        raise TokenAuth::Concerns::ApiResources::Api::Unauthorized
```

Comment:
Slight nuance reported [here](http://stackoverflow.com/questions/31937632/fail-vs-raise-in-ruby-should-we-really-believe-the-style-guide):

> use 'raise' for exceptions to be caught, and 'fail' for serious errors which are not meant to be handled